assets/apps/dashboard/src/Components/App.js

```diff
@@ -30,9 +30,10 @@ const App = ({ setSettings, toast, currentTab, setTab }) => {
 						{'starter-sites' !== currentTab && <Notifications />}
 						<TabsContent currentTab={currentTab} setTab={setTab} />
 					</div>
-					{'starter-sites' !== currentTab && (
-						<Sidebar currentTab={currentTab} />
-					)}
+					{'starter-sites' !== currentTab &&
+						'custom-layouts' !== currentTab && (
+							<Sidebar currentTab={currentTab} />
+						)}
```

Comment:
When using multiple conditions with &&, you should wrap them in brackets as it might result in unexpected output of 0/1 in the UI.

```suggestion
					{ ( 'starter-sites' !== currentTab && 'custom-layouts' !== currentTab ) && (
						<Sidebar currentTab={currentTab} />
					)}
```

Reply:
I don't think it applies here as it the results are always boolean and it is not a `falsy` condition. Also, the eslint will remove the extra wrapping brackets.